docs/topics_5/web-analyze-apps.adoc

```diff
@@ -3,17 +3,11 @@
 [id='using_web_console_analyze_apps_{context}']
 = Analyzing applications with the {WebName}
 
```

Comment:
This is not a module. It is a title with an introduction.

Move the header and the FIRST paragraph to master.adoc.  I am going to suggest a rewording for the first paragraph because I don't like beginning an introduction with a subordinate clause. It's too easy for the user not to read it.



Remove "Follow these steps".